src/svg-quirks-mode/fonts/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
@tmickel Unfortunately I think a repo just dedicated to the fonts would be best.